app/src/main/java/org/jellyfin/androidtv/ui/ClockUserView.java

```diff
@@ -32,6 +38,33 @@ private void init(Context context) {
         LayoutInflater inflater = LayoutInflater.from(context);
         View v = inflater.inflate(R.layout.clock_user_bug, null, false);
         this.addView(v);
+        TextClock tc = v.findViewById(R.id.clock);
+        ClockBehavior showClock = get(UserPreferences.class).get(UserPreferences.Companion.getShowClock());
+        if (showClock != ClockBehavior.ALWAYS) {
```

Comment:
The combination of this if-statement and the switch-statement for all the `setVisibility` can be simplified for better readability.